src/standalone/Soil/rre.jl

```diff
@@ -61,18 +63,21 @@ end
         domain::D,
         boundary_conditions::NamedTuple,
         sources::Tuple,
+        lateral_flow::Bool = true
     ) where {FT, D}
 
-A constructor for a `RichardsModel`.
+A constructor for a `RichardsModel`, which sets the 
+default value of `lateral_flow` to be true.
 """
 function RichardsModel{FT}(;
     parameters::RichardsParameters{FT},
     domain::D,
     boundary_conditions::NamedTuple,
     sources::Tuple,
+    lateral_flow::Bool = true,
 ) where {FT, D}
     args = (parameters, domain, boundary_conditions, sources)
-    RichardsModel{FT, typeof.(args)...}(args...)
+    RichardsModel{FT, typeof.(args)...}(args..., lateral_flow)
```

Comment:
Same comment as in energy_hydrology.jl - maybe `lateral_flow` can be in the `args` tuple

Reply:
same as above. happy to implement something cleaner!